src/components/HeadlineArea/HeadlineArea.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react'
+import styled, { css } from 'styled-components'
+import { InjectedProps, withTheme } from '../../hocs/withTheme'
+import { HeadingProps, Heading } from '../Heading/Heading'
+
+interface Props {
+  heading: HeadingProps
+  description?: React.ReactNode
+}
+
+const HeadlineAreaComponent: React.FC<Props & InjectedProps> = ({ theme, ...props }) => (
+  <Wrapper theme={theme}>
+    <Heading type="ScreenTitle" tag={props.heading.tag ? props.heading.tag : 'h1'}>
```

Comment:
Since type of heading is set to HeadingProps, the user may pass heading.type props.
However, whatever you give to heading.type props, it becomes ScreenTitle.
This behavior is not intuitive.

It is better to exclude heading.props.

```tsx
interface Props {
  heading: {
    children: HeadingProps['children']
    tag?: HeadingProps['tag']
  }
  description?: React.ReactNode
}
```